src/services/SessionProcessor.ts

```diff
@@ -125,11 +125,14 @@ export class SessionProcessor {
 			const sub = jwtIdTokenPayload.sub!;
 			try {
 				session = await iprService.getSessionBySub(sub);
-				this.logger.debug("Session retrieved from session store");
 				if (!session) {
 					this.logger.error("No session event found for this userId");
 					return new Response(HttpCodesEnum.UNAUTHORIZED, "No session event found for this userId");
 				}
+				this.logger.appendKeys({
+					govuk_signin_journey_id: session.clientSessionId,
```

Comment:
I think this `govuk_signin_journey_id` would be a new one and needs to be extracted from the JWT